lib/deep-readonly/index.ts

```diff
@@ -21,10 +21,22 @@ export type DeepReadonly<Type> = Type extends Exclude<Builtin, Error>
   ? Promise<DeepReadonly<Value>>
   : Type extends AnyArray<infer Values>
```

Comment:
Can you add changelog using `yarn changeset`?

This is documented in https://github.com/ts-essentials/ts-essentials/blob/master/CONTRIBUTING.md. Let me know if anything is unclear